tests/trainer/optimization/test_manual_optimization.py

```diff
@@ -957,6 +957,59 @@ def configure_optimizers(self):
     trainer.fit(model)
 
 
+@pytest.mark.parametrize("scheduler_as_dict", [True, False])
+def test_lr_schedulers_reduce_lr_on_plateau(tmpdir, scheduler_as_dict):
+    class TestModel(BoringModel):
+        def __init__(self, scheduler_as_dict):
+            super().__init__()
+            self.scheduler_as_dict = scheduler_as_dict
+            self.automatic_optimization = False
+
+        def training_step(self, batch, batch_idx):
+            return {"train_loss_1": torch.tensor([0.0]), "train_loss_2": torch.tensor([0.0])}
+
+        def training_epoch_end(self, outputs):
+            scheduler_1, scheduler_2 = self.lr_schedulers()
+
+            loss = torch.stack([x["train_loss_1"] for x in outputs]).mean()
+            scheduler_1.step(loss)
+
+            loss = torch.stack([x["train_loss_2"] for x in outputs]).mean()
+            scheduler_2.step(loss)
+
+        def configure_optimizers(self):
```

Comment:
are two schedulers required here? we already have tests with muliple schedulers + manual right?

Reply:
Not really. Updated!